### PR TITLE
fix(parser): unexpected ts type annotation in get/set

### LINKED ISSUE #1939: Syntax error in firefox, but not in oxc
```js
class A {
  get x: () => {
    return 5;
  }
}
```

### PR DESCRIPTION
**Pull request by Dunqing**
fix: https://github.com/oxc-project/oxc/issues/1939

### CODE REVIEW

**Comment by Dunqing** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1942** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1942?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/1942?utm_source=stack-comment).

File: crates/oxc_parser/src/js/class.rs [hunk 1]
```diff
@@ -294,6 +294,10 @@ impl<'a> Parser<'a> {
             }
             Ok(definition)
         } else {
+            // get / set no ts type annotation
```
Comment (Boshen) [inline crates/oxc_parser/src/js/class.rs:297]:
```suggestion
            // getter and setter has no ts type annotation
```

File: crates/oxc_parser/src/lib.rs [hunk 2]
```diff
@@ -330,6 +330,34 @@ mod test {
         assert_eq!(ret.errors.first().unwrap().to_string(), "Source length exceeds 4 GiB limit");
     }
 
+    #[test]
```
Comment (Boshen) [inline crates/oxc_parser/src/lib.rs:333]: Add then here: https://github.com/oxc-project/oxc/tree/main/tasks/coverage/misc/pass

We don't need to bloat this file with syntax tests

Comment (Dunqing) [reply crates/oxc_parser/src/lib.rs]: That's what I want to know

**Review by Boshen — APPROVED** [review]

**Pull request merged**